docs/invoicecapture/documentExport.md

```diff
@@ -0,0 +1,248 @@
+
+The response sections included below all assume the web service call is successful, returning an HTTP 200 code. Requests which are unauthorised will return a 401 error code, all other unsuccessful web service calls will return a 400 error code along with a detailed description of the error.
+
+---
+
+##Authentication
+
+Authentication with Invoice Capture is performed with the use of an API key. This key is a one-time generated key, which can be used by the third party to generate an access token prior to each API call.  Access tokens are valid for up to 3 hours, but expire after 10 minutes of inactivity.
+
+You can get the API for your Invoice Capture instance by contacting Proactis support.
+
+All URLs given are relative to the root of your Invoice Capture instance.
+
+###Request
+
+**URL**
+
+| Method | URL |
+|--------|-----|
+| **GET** | **/API/ExportStagingArea/Account/GetAccessToken** |
+
+
+**Parameters**
+
+| Mandatory | Parameter | Value |
+|----------|-----|-----------|
+| &#10004; | apiKey | API key from Proactis |
+
+###Response
+
+The web service will return an access token which can be used to call the other web services.
+
+```xml
+"54ece54e-98c2-432d-9c0d-b098e80f631d"
+```
+
+---
+
+##Get Available Formats
+
+Documents may be exported in a number of different formats. The GetAvailableFormats web service returns the formats available to your Invoice Capture instance. 
+
+The format type is required when calling the Documents export web service
+
+###Request
+
+**URL**
+
+| Method | URL |
+|--------|-----|
+| **GET** | **/API/ExportStagingArea/Documents/AvailableFormats** |
+
+**Parameters**
+
+| Mandatory | Parameter | Value |
+|----------|-----|-----------|
+| &#10004; | Token | The access token returned by the GetAccessToken web service |
+
+
+###Response
+
+The list of available formats is returned as a string array.
+
+```json
+[
+    "JSON Invoice"
+]
+```
+
+---
+
+## Get Documents to Process
+
+The ReferenceData web service returns a list of documents from Invoice Capture based on the parameters passed. You can search for all documents or just those pending processing and filter based on the date they were exported from Invoice Capture.
+
+###Request
+
+**URL**
+
+| Method | URL |
+|--------|-----|
+| **GET** | **/API/ExportStagingArea/Documents/ReferenceData** |
+
+**Parameters**
+
+| Mandatory | Parameter | Value |
+|----------|-----------|-------|
+| &#10004; | Token | The access token returned by the GetAccessToken web service |
+| &#10004; | State | Valid values are All or Pending. **All** returns all documents. **Pending** returns those which have not yet been confirmed as exported. |
+| | StartDate | Optional. Only returns documents where the exported date is greater than or equal to the supplied date. Date must supplied in the format yyyy-MM-dd. |
+| | EndDate | Optional. Only returns documents where the exported date is less than or equal to the supplied date. Date must supplied in the format yyyy-MM-dd. |
+
```

Comment:
Might be worth adding an example call,  with the query string parameters populated.